ibis/backends/polars/compiler.py

```diff
@@ -244,7 +244,9 @@ def fillna(op):
             value = op.replacements.value
 
         if value is not None:
-            column = column.fill_nan(value).fill_null(value)
+            if isinstance(dtype, dt.Floating):
```

Comment:
Did you mean to include this in this PR, or should this be a separate change?

Reply:
I meant to include it, because it's a change necessary for the latest polars version.